README.md

```diff
@@ -1,2 +1,34 @@
-# graphql-logger
-A library that organizes graphql logs
+![Small Logo](static/img/polaris-logo.png)
+
+# polaris-graphql-logger
+
+[![Build Status](https://travis-ci.com/Enigmatis/polaris-graphql-logger.svg?branch=master)](https://travis-ci.com/Enigmatis/polaris-graphql-logger)
+[![NPM version](https://img.shields.io/npm/v/@enigmatis/polaris-graphql-logger.svg?style=flat-square)](https://www.npmjs.com/package/@enigmatis/polaris-graphql-logger)
+
+#### Install
+
+```
+npm install polaris-graphql-logger
+```
+
+### Overview
+
+This library organizes and defines everything that wrapped around using a classic graphql logger.
```

Comment:
Polaris GraphQL Logger allows you to write standardized logs easily, using a polaris-logs logger, and wrapping a classic graphql logger.